extensions/collection-assets/json-schema/schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema.json#",
+  "title": "Collection Assets Extension Specification",
+  "description": "STAC Collection-level assets Extension to a STAC Collection",
+  "allOf": [
+    {
+      "$ref": "../../../collection-spec/json-schema/collection.json"
+    },
+    {
+      "type": "object",
+      "required": [
+        "assets"
+      ],
+      "properties": {
+        "assets": {
+          "$ref": "../../../item-spec/json-schema/item.json#/definitions/assets"
```

Comment:
Since assets would no longer be an entity definition owned by items, would it make sense to migrate the "Asset Object" section into its own specification? Or is the rationale that, since this is an extension, asset objects actually are owned by items still, unless this extension is present on the collection?

Reply:
Well, we have quite a lot of things that are shared across spec (Provider Object, Asset Object, Link Object, ...). I think we could move if it's used in core specs, but extensions shouldn't influence where we put things in the core spec. So would leave it as it is for now.